template/dockerfiles/python/Dockerfile

```diff
@@ -9,4 +9,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 COPY . .
 
 ENTRYPOINT ["python"]
-CMD ["app.py"]
\ No newline at end of file
+CMD ["{{ENTRYPOINT}}"]
```

Comment:
since we are now using the ENTRYPOINT draft variable, we should add it to the corresponding [draft.yaml](https://github.com/Azure/draft/blob/762261493f50c839ec240b41f3a3ab42a7c31756/template/dockerfiles/python/draft.yaml) too, with a sane default (maybe just keep app.py or main.py for the draft.yaml's default)

Reply:
added ENTRYPOINT variable with default app.py